_docs/sandbox-testing.md

```diff
@@ -14,6 +14,122 @@ To facilitate testing our sandbox environment has a few features that can help i
+```vb
+Dim payinMethodRequest as PayinMethodRequest = New PayinMethodRequest(type:="GHS::Mobile")
+Dim api as TransactionsApi = New TransactionsApi(configuration)
+Try
+  Dim response as TransactionResponse = api.PayinTransaction("77f89fa-8371-496c-ae73-e56697fc08d8", payinMethodRequest)
+Catch e As ApiException
+  If e.IsValidationError Then
+    Dim response as TransactionResponse = e.ParseObject(Of TransactionResponse)()
+    ' Process validation error
+  End If
+  Throw e
+End Try
+```
+{% endcapture %}
+
+{% capture data-java %}
+```java
+PayinMethodRequest) payinMethodRequest = new PayinMethodRequest();
+payin_method_request.setType("GHS::Mobile");
+TransactionsApi) api = new TransactionsApi(apiClient);
+try {
+  TransactionResponse response = api.payinTransaction("77f89fa-8371-496c-ae73-e56697fc08d8", payinMethodRequest);
+} catch (ApiException e) {
+  if (e.isValidationError()) {
+    TransactionResponse response = e.getResponseObject(TransactionResponse.class);
+    // Process validation error
+  }
+  throw e;
+};
+```
+{% endcapture %}
+
+{% capture data-js %}
+```js
+const payinMethodRequest = new TransferZeroSdk.PayinMethodRequest();
+payinMethodRequest.type = "GHS::Mobile";
+const api = new TransferZeroSdk.TransactionsApi(apiClient);
+try {
+  const response = await api.payinTransaction("77f89fa-8371-496c-ae73-e56697fc08d8", payinMethodRequest);
+} catch (e) {
+  if (e.isValidationError) {
+    const response = e.getResponseObject();
+    // Process validation error
+  }
+  throw e;
+}
+```
+{% endcapture %}
+
+{% capture data-php %}
+```php
+$payinMethodRequest = new PayinMethodRequest();
+$payinMethodRequest->setType("GHS::Mobile");
+$api = new TransactionsApi();
+try {
+  $response = api->payinTransaction("77f89fa-8371-496c-ae73-e56697fc08d8", payinMethodRequest);
+} catch (ApiException $e) {
+  if ($e->isValidationError()) {
+    $response = $e->getResponseObject();
+    // Process validation error
+  }
+  throw $e;
+}
+```
+{% endcapture %}
+
+{% capture data-ruby %}
+```ruby
+payin_method_request = TransferZero::PayinMethodRequest.new
+payin_method_request.type = "GHS::Mobile"
+api = TransferZero::TransactionsApi.new
+begin
+  response = api.payin_transaction("77f89fa-8371-496c-ae73-e56697fc08d8", payin_method_request)
+rescue TransferZero::ApiError => e
+  if e.validation_error
+    response = e.response_object("TransactionResponse")
+    # Process validation error
+  end
+  raise e
+end
+```
+{% endcapture %}
+{% include language-tabbar.html prefix="gen-payin-transaction-call" raw=data-raw csharp=data-csharp vb=data-vb java=data-java js=data-js php=data-php ruby=data-ruby %}
+
 # Forcing collections to succeed
 
 When using our [collections feature]({{ "/docs/additional-features/" | prepend: site.baseurl }}#collections-from-senders) you can force the collection to succeed. Please see the documentation of this feature in our [API specification guide](https://api.transferzero.com/documentation/#/Transactions/payin-transaction)
```

Comment:
Should we also remove this, or incorporate it into the additions you've made?

Reply:
Thanks